google/cloud/bigtable/doc/bigtable-samples.dox

```diff
@@ -0,0 +1,60 @@
+/*!
+@page bigtable-samples Example: Table Admin Samples
+
+This example is a very simple "Table Admin" application, which illustrates how to:
+
+- Create a table and run basic table admin operations.
+- Create a table.
+- List tables.
+- Retrieve table metadata
+- Modify the existing tables
+- Delete all the rows.
+- Delete a table.
+
+## Run the example
+
+This example uses the
+[Cloud Bigtable C++ Client Library](https://GoogleCloudPlatform.github.io/google-cloud-cpp)
+to communicate with Cloud Bigtable.
+
+To run the example program, follow the
+[instructions](https://github.com/GoogleCloudPlatform/google-cloud-cpp/tree/master/google/cloud/bigtable/examples/)
+for the example on GitHub.
+
+### Include the Necessary Headers
+
+The example uses the following headers:
+
+@snippet bigtable_samples.cc bigtable includes
```

Comment:
the `bigtable includes` marker does not exist in bigtable_samples.cc that broke the build for you.